backend/api/Controllers/ReportController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ReportController : ControllerBase
+{
+    private readonly ILogger<ReportController> _logger;
+    private readonly ReportService _reportService;
+
+    public ReportController(ILogger<ReportController> logger, ReportService reportService)
+    {
+        _logger = logger;
+        _reportService = reportService;
+    }
+
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<Report>>> GetReports()
```

Comment:
Consider adding status code attributes similar to the robot controller

Reply:
Agreed, forgot about this, will fix